unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -41,9 +41,9 @@ export class CreatorLockForm extends React.Component {
       keyPrice = Web3Utils.fromWei(keyPrice, props.keyPriceCurrency)
       expirationDuration = expirationDuration / props.expirationDurationUnit
       // Unlimited keys is represented as zero, so when we load a lock that
```

Comment:
There's still a stray zero in the comment here, we can probably rephrase entirely to "unlimited keys is represented differently in the frontend and the backend, so when a lock has `maxNumberOfKeys` set to UNLIMITED_KEYS_COUNT, convert it to infinity symbol for frontend display"